agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -551,8 +551,8 @@ case ${key} in
     -op|--openshift)
         OPENSHIFT=1
         ;;
-    -as|--agent-slim)
```

Comment:
Why do we need to change the interface for this script? Can't we leave `-as` for backwards compatibility?

Reply:
Can this option be left, send a `this will be deprecated` message and set `AGENT_FULL=1`?

Reply:
Comments from @0snug0 
https://sysdig.atlassian.net/browse/SMAGENT-2906?focusedCommentId=152218

* Ok to delete comments
* @0snug0 @mbreitung Can any of you confirm if missed init container limit for cpu should be equal to request (1000m)